### DISCUSSION TITLE
Response Flagging?

### DISCUSSION THREAD

**Discussion by tedraymond** [open-webui/open-webui#2178] [General]
First, awesome tool. Installed and up and running in no time today.

Question though, when a user clicks the thumb up or thumb down, where does that flagging save too?

Pretty new to web ui, in gradio I'd manually define the location.

Any thoughts?

**Comment by justinh-rahb** [comment]
Currently it's only saved to the database with the chats. Intended use currently is up to you, you could in theory use it for RLHF dataset. Future use would be doing the same, but within our project :)

**Reply by tedraymond** [reply]
Ah, i like it. Thanks for the information.

Nice work.
